src/status_im2/contexts/chat/messages/content/pin/view.cljs

```diff
@@ -23,40 +23,41 @@
        :style  style/pin-author-text}
       author-name]]))
 
+(def system-message-default-size 36)
+(def system-message-radius (/ system-message-default-size 2))
+
 (defn system-message
-  [{:keys [from in-popover? timestamp-str chat-id] :as message}]
-  (let [response-to  (:response-to (:content message))
-        default-size 36]
-    [rn/touchable-opacity
-     {:on-press       (fn []
-                        (rf/dispatch [:dismiss-keyboard])
-                        (rf/dispatch [:pin-message/show-pins-bottom-sheet chat-id]))
-      :active-opacity 1
-      :style          (merge {:flex-direction :row :margin-vertical 8}
-                             (old-style/message-wrapper message))}
-     [rn/view
-      {:style               {:width            default-size
-                             :height           default-size
-                             :margin-right     16
-                             :border-radius    default-size
-                             :justify-content  :center
-                             :align-items      :center
-                             :background-color colors/primary-50-opa-10}
-       :accessibility-label :content-type-pin-icon}
-      [quo/icon :i/pin {:color colors/primary-50 :size 16}]]
-     [rn/view
-      [rn/view {:style {:flex-direction :row :align-items :center}}
-       [rn/touchable-opacity
-        {:style    old-style/message-author-touchable
-         :disabled in-popover?
-         :on-press #(rf/dispatch [:chat.ui/show-profile from])}
-        [old-message/message-author-name from {} 20]]
-       [rn/text {:style {:font-size 13}} (str " " (i18n/label :t/pinned-a-message))]
-       [rn/text
-        {:style               (merge
-                               {:padding-left 5
-                                :margin-top   2}
-                               (old-style/message-timestamp-text))
-         :accessibility-label :message-timestamp}
-        timestamp-str]]
-      [old-message/quoted-message {:message-id response-to :chat-id chat-id} true]]]))
+  [{:keys [from in-popover? quoted-message timestamp-str chat-id] :as message}]
+  [rn/touchable-opacity
+   {:on-press       (fn []
+                      (rf/dispatch [:dismiss-keyboard])
+                      (rf/dispatch [:pin-message/show-pins-bottom-sheet chat-id]))
+    :active-opacity 1
+    :style          (merge {:flex-direction :row :margin-vertical 8}
```

Comment:
can we put these styles in a style.cljs file?

i.e `{:flex-direction :row :margin-vertical 8}`